spec/models/user_spec.rb

```diff
@@ -453,11 +453,11 @@
     end
   end
 
-  describe '#process_invite_acceptence' do
+  describe '#process_invite_acceptance' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.